classes/WP_CLI/ActionScheduler_WPCLI_Migration_Command.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+
+namespace Action_Scheduler\WP_CLI;
+
+use Action_Scheduler\Migration\ActionScheduler_MigrationConfig;
+use Action_Scheduler\Migration\ActionScheduler_MigrationRunner;
+use Action_Scheduler\Migration\ActionScheduler_MigrationScheduler;
+use ActionScheduler_Data;
+use WP_CLI;
+use WP_CLI_Command;
+
+/**
+ * Class ActionScheduler_WPCLI_Migration_Command
+ *
+ * @package Action_Scheduler\WP_CLI
+ *
+ * @codeCoverageIgnore
+ */
+class ActionScheduler_WPCLI_Migration_Command extends WP_CLI_Command {
+
+	/**
+	 * Migrates actions to the DB tables store
+	 */
+
+	private $total_processed = 0;
+
+	/**
+	 * Register the command with WP-CLI
+	 */
+	public function register() {
+		if ( ! defined( 'WP_CLI' ) || ! WP_CLI ) {
+			return;
+		}
+
+		WP_CLI::add_command( 'action-scheduler db-tables migrate', [ $this, 'migrate' ], [
```

Comment:
The `db-tables` component of this command feels unnecessary now that the code is in AS core. What about just `action-scheduler migrate`? (Or a similar keyword used by WC if it differs)